src/pages/SwapV3/index.tsx

```diff
@@ -0,0 +1,567 @@
+  useShowTokenInfo,
+  useShowTradeRoutes,
+  useToggleProLiveChart,
+  useUserAddedTokens,
+  useUserSlippageTolerance,
+} from 'state/user/hooks'
+import { RouteSummary } from 'types/metaAggregator'
+import { getTradeCompositionForRouteSummary } from 'utils/aggregationRouting'
+import { currencyId } from 'utils/currencyId'
+import { getSymbolSlug } from 'utils/string'
+import { checkPairInWhiteList } from 'utils/tokenInfo'
+
+import PopulatedSwapForm from './PopulatedSwapForm'
+
+const TradeRouting = lazy(() => import('../../components/TradeRouting'))
+const LiveChart = lazy(() => import('components/LiveChart'))
+
+const TutorialIcon = styled(TutorialSvg)`
+  width: 22px;
+  height: 22px;
+  path {
+    fill: ${({ theme }) => theme.subText};
+    stroke: ${({ theme }) => theme.subText};
+  }
+`
+
+enum TAB {
+  SWAP = 'swap',
+  INFO = 'info',
+  SETTINGS = 'settings',
+  GAS_PRICE_TRACKER = 'gas_price_tracker',
+  LIQUIDITY_SOURCES = 'liquidity_sources',
+  LIMIT = 'limit',
+}
+
+const highlight = (theme: DefaultTheme) => keyframes`
+  0% {
+    box-shadow: 0 0 0 0 ${theme.primary};
+  }
+
+  70% {
+    box-shadow: 0 0 0 2px ${theme.primary};
+  }
+
+  100% {
+    box-shadow: 0 0 0 0 ${theme.primary};
+  }
+`
+
+const AppBodyWrapped = styled(BodyWrapper)`
+  box-shadow: 0px 4px 16px rgba(0, 0, 0, 0.04);
+  padding: 16px;
+  margin-top: 0;
+
+  &[data-highlight='true'] {
+    animation: ${({ theme }) => highlight(theme)} 2s 2 alternate ease-in-out;
+  }
+`
+
+const SwitchLocaleLinkWrapper = styled.div`
+  margin-bottom: 30px;
+  ${({ theme }) => theme.mediaWidth.upToMedium`
+  margin-bottom: 0px;
+`}
+`
+
+const RoutingIconWrapper = styled(RoutingIcon)`
+  height: 27px;
+  width: 27px;
+  margin-right: 10px;
+  path {
+    fill: ${({ theme }) => theme.subText} !important;
+  }
+`
+
+export default function Swap() {
+  const navigateFn = useNavigate()
+  const { chainId, networkInfo, isSolana } = useActiveWeb3React()
+  const isShowLiveChart = useShowLiveChart()
+  const toggleProLiveChart = useToggleProLiveChart()
+  const isShowTradeRoutes = useShowTradeRoutes()
+  const isShowTokenInfoSetting = useShowTokenInfo()
+  const qs = useParsedQueryString<{
+    highlightBox: string
+    outputCurrency: string
+    inputCurrency: string
+  }>()
+  const [{ show: isShowTutorial = false }] = useTutorialSwapGuide()
+  const [routeSummary, setRouteSummary] = useState<RouteSummary>()
+
+  const { pathname } = useLocation()
+  useSyncNetworkParamWithStore()
+
+  const refSuggestPair = useRef<PairSuggestionHandle>(null)
+  const refListLimitOrder = useRef<ListOrderHandle>(null)
+
+  const [showingPairSuggestionImport, setShowingPairSuggestionImport] = useState<boolean>(false) // show modal import when click pair suggestion
+
+  const shouldHighlightSwapBox = qs.highlightBox === 'true'
+
+  const [isSelectCurrencyManually, setIsSelectCurrencyManually] = useState(false) // true when: select token input, output manualy or click rotate token.
```

Comment:
move it into LimitOrder component

Reply:
I tried but it affects several places. So I think it's OK for now to keep it here.